pkg/services/ngalert/notifier/channels/pagerduty.go

```diff
@@ -88,7 +92,19 @@ func buildPagerdutySettings(fc FactoryConfig) (*pagerdutySettings, error) {
 	if settings.Summary == "" {
 		settings.Summary = DefaultMessageTitleEmbed
 	}
-
+	if settings.Client == "" {
+		settings.Client = defaultClient
+	}
+	if settings.ClientURL == "" {
+		settings.ClientURL = "{{ .ExternalURL }}"
+	}
+	if settings.Source == "" {
+		source, err := os.Hostname()
```

Comment:
Source is supposed to be the system that's affected by the firing alert - that's almost never going to be the machine running Grafana itself, but rather, whatever Grafana is monitoring. I'm not sure if hostname is the right default here.

Should the default simply be blank/not present?

Reply:
I made this for backward compatibility. I do not have a strong opinion, though.

Reply:
I think we can remove it later as part of the unificiation process